### PR TITLE
Change setting of jenkins trigger

### PR DESCRIPTION
**Pull request by j-mracek**
This patch enable job's trigger only to members of white list

### CODE REVIEW

File: jobs/github-testers-macros.yaml [hunk 1]
```diff
@@ -44,12 +44,20 @@
       - github-pull-request:
           auth-id: "{ghprb-auth-id}"
 
+          white-list:
```
Comment (ignatenkobrain) [inline jobs/github-testers-macros.yaml]: white-list should be defined per-project.

Comment (j-mracek) [inline jobs/github-testers-macros.yaml]: Because there are different member list?

Comment (ignatenkobrain) [inline jobs/github-testers-macros.yaml]: Exactly

**Comment by mluscon** [comment]
```
INFO:root:Updating jobs in ['.'] ([])
INFO:jenkins_jobs.local_yaml:Including file 'github-token.txt' from path '.'
INFO:jenkins_jobs.local_yaml:Including file 'ghprb-auth-id.txt' from path '.'
INFO:jenkins_jobs.local_yaml:Including file 'github-token.txt' from path '.'
INFO:jenkins_jobs.local_yaml:Including file 'ghprb-auth-id.txt' from path '.'
INFO:jenkins_jobs.local_yaml:Including file 'github-token.txt' from path '.'
INFO:jenkins_jobs.local_yaml:Including file 'ghprb-auth-id.txt' from path '.'
INFO:jenkins_jobs.local_yaml:Including file 'github-token.txt' from path '.'
INFO:jenkins_jobs.local_yaml:Including file 'ghprb-auth-id.txt' from path '.'
INFO:jenkins_jobs.local_yaml:Including file 'github-token.txt' from path '.'
INFO:jenkins_jobs.local_yaml:Including file 'ghprb-auth-id.txt' from path '.'
INFO:jenkins_jobs.local_yaml:Including file 'github-token.txt' from path '.'
INFO:jenkins_jobs.local_yaml:Including file 'ghprb-auth-id.txt' from path '.'
Traceback (most recent call last):
  File "/usr/lib/python3.5/site-packages/jenkins_jobs/formatter.py", line 40, in deep_format
    ret = CustomFormatter(allow_empty).format(obj, **paramdict)
  File "/usr/lib64/python3.5/string.py", line 187, in format
    return self.vformat(format_string, args, kwargs)
  File "/usr/lib64/python3.5/string.py", line 191, in vformat
    result, _ = self._vformat(format_string, args, kwargs, used_args, 2)
  File "/usr/lib64/python3.5/string.py", line 231, in _vformat
    obj, arg_used = self.get_field(field_name, args, kwargs)
  File "/usr/lib64/python3.5/string.py", line 296, in get_field
    obj = self.get_value(first, args, kwargs)
  File "/usr/lib/python3.5/site-packages/jenkins_jobs/formatter.py", line 77, in get_value
    return Formatter.get_value(self, key, args, kwargs)
  File "/usr/lib64/python3.5/string.py", line 253, in get_value
    return kwargs[key]
KeyError: 'white-list'

```

**Comment by j-mracek** [comment]
@mluscon I created fixup and tested by deploy:
jenkins-jobs --conf=`pwd`/jobs/jenkins_jobs.ini update jobs/
INFO:root:Updating jobs in ['jobs/'] ([])
INFO:jenkins_jobs.local_yaml:Including file 'github-token.txt' from path '/home/jmracek/Projects/ci-dnf-stack/jobs'
INFO:jenkins_jobs.local_yaml:Including file 'ghprb-auth-id.txt' from path '/home/jmracek/Projects/ci-dnf-stack/jobs'
INFO:jenkins_jobs.local_yaml:Including file 'github-token.txt' from path '/home/jmracek/Projects/ci-dnf-stack/jobs'
INFO:jenkins_jobs.local_yaml:Including file 'ghprb-auth-id.txt' from path '/home/jmracek/Projects/ci-dnf-stack/jobs'
INFO:jenkins_jobs.local_yaml:Including file 'github-token.txt' from path '/home/jmracek/Projects/ci-dnf-stack/jobs'
INFO:jenkins_jobs.local_yaml:Including file 'ghprb-auth-id.txt' from path '/home/jmracek/Projects/ci-dnf-stack/jobs'
INFO:jenkins_jobs.local_yaml:Including file 'github-token.txt' from path '/home/jmracek/Projects/ci-dnf-stack/jobs'
INFO:jenkins_jobs.local_yaml:Including file 'ghprb-auth-id.txt' from path '/home/jmracek/Projects/ci-dnf-stack/jobs'
INFO:jenkins_jobs.local_yaml:Including file 'github-token.txt' from path '/home/jmracek/Projects/ci-dnf-stack/jobs'
INFO:jenkins_jobs.local_yaml:Including file 'ghprb-auth-id.txt' from path '/home/jmracek/Projects/ci-dnf-stack/jobs'
INFO:jenkins_jobs.builder:Number of jobs generated:  39
INFO:jenkins_jobs.builder:Reconfiguring jenkins job dnf-build
INFO:jenkins_jobs.builder:Reconfiguring jenkins job dnf-plugins-core-build
INFO:jenkins_jobs.builder:Reconfiguring jenkins job dnf-plugins-core-provision
INFO:jenkins_jobs.builder:Reconfiguring jenkins job dnf-plugins-core-teardown
INFO:jenkins_jobs.builder:Reconfiguring jenkins job dnf-plugins-core-test
INFO:jenkins_jobs.builder:Reconfiguring jenkins job dnf-plugins-core-trigger
INFO:jenkins_jobs.builder:Reconfiguring jenkins job dnf-plugins-core-update-gh-status
INFO:jenkins_jobs.builder:Reconfiguring jenkins job dnf-provision
INFO:jenkins_jobs.builder:Reconfiguring jenkins job dnf-teardown
INFO:jenkins_jobs.builder:Reconfiguring jenkins job dnf-test
INFO:jenkins_jobs.builder:Reconfiguring jenkins job dnf-trigger
INFO:jenkins_jobs.builder:Reconfiguring jenkins job dnf-update-gh-status
INFO:jenkins_jobs.builder:Reconfiguring jenkins job libdnf-build
INFO:jenkins_jobs.builder:Reconfiguring jenkins job libdnf-provision
INFO:jenkins_jobs.builder:Reconfiguring jenkins job libdnf-teardown
INFO:jenkins_jobs.builder:Reconfiguring jenkins job libdnf-test
INFO:jenkins_jobs.builder:Reconfiguring jenkins job libdnf-trigger
INFO:jenkins_jobs.builder:Reconfiguring jenkins job libdnf-update-gh-status
INFO:jenkins_jobs.builder:Reconfiguring jenkins job libsolv-build
INFO:jenkins_jobs.builder:Reconfiguring jenkins job libsolv-provision
INFO:jenkins_jobs.builder:Reconfiguring jenkins job libsolv-teardown
INFO:jenkins_jobs.builder:Reconfiguring jenkins job libsolv-test
INFO:jenkins_jobs.builder:Reconfiguring jenkins job libsolv-trigger
INFO:jenkins_jobs.builder:Reconfiguring jenkins job libsolv-update-gh-status
INFO:jenkins_jobs.builder:Reconfiguring jenkins job nightly-epel-7-x86_64
INFO:jenkins_jobs.builder:Reconfiguring jenkins job nightly-fedora-23-i386
INFO:jenkins_jobs.builder:Reconfiguring jenkins job nightly-fedora-23-x86_64
INFO:jenkins_jobs.builder:Reconfiguring jenkins job nightly-fedora-24-i386
INFO:jenkins_jobs.builder:Reconfiguring jenkins job nightly-fedora-24-x86_64
INFO:jenkins_jobs.builder:Reconfiguring jenkins job nightly-fedora-25-i386
INFO:jenkins_jobs.builder:Reconfiguring jenkins job nightly-fedora-25-x86_64
INFO:jenkins_jobs.builder:Reconfiguring jenkins job nightly-fedora-rawhide-i386
INFO:jenkins_jobs.builder:Reconfiguring jenkins job nightly-fedora-rawhide-x86_64
INFO:jenkins_jobs.builder:Reconfiguring jenkins job rpm-build
INFO:jenkins_jobs.builder:Reconfiguring jenkins job rpm-provision
INFO:jenkins_jobs.builder:Reconfiguring jenkins job rpm-teardown
INFO:jenkins_jobs.builder:Reconfiguring jenkins job rpm-test
INFO:jenkins_jobs.builder:Reconfiguring jenkins job rpm-trigger
INFO:jenkins_jobs.builder:Reconfiguring jenkins job rpm-update-gh-status
INFO:root:Number of jobs updated: 39
INFO:jenkins_jobs.builder:Cache saved

**Comment by ignatenkobrain** [comment]
Testing by deploy.... sounds wrong ;)

**Comment by j-mracek** [comment]
@ignatenkobrain You all right, but how you would prefer to test it?

**Comment by j-mracek** [comment]
@mluscon I think now it should work.

**Comment by j-mracek** [comment]
@dnf-bot r+

**Comment by j-mracek** [comment]
merged

**Pull request closed without merging**